app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -29,7 +29,13 @@ class DragAndDropSortInteractionViewModel(
     interaction.customizationArgsMap["allowMultipleItemsInSamePosition"]?.boolValue ?: false
   }
   private val choiceStrings: List<String> by lazy {
-    interaction.customizationArgsMap["choices"]?.setOfHtmlString?.htmlList ?: listOf()
+    val choicesList = mutableListOf<String>()
+    val schemaObjectList =
+      interaction.customizationArgsMap["choices"]?.schemaObjectList?.schemaObjectList
+    schemaObjectList?.forEach { schemaObject ->
+      choicesList.add(schemaObject.subtitledHtml.html)
+    }
+    choicesList
```

Comment:
```suggestion
    interaction.customizationArgsMap["choices"]
        ?.schemaObjectList
        ?.schemaObjectList
        ?.map { schemaObject -> schemaObject.subtitledHtml.html }
        ?: listOf()
```

This is a bit more idiomatic since it doesn't involve mutating a list, and instead computes it step-by-step or early exits to an empty list.

Reply:
Updated. Thanks.